### PR TITLE
[RFR] Redesign with curses

### PR DESCRIPTION
**Pull request by HALLERPierre**
[Trello](https://trello.com/c/nS8r7mOg/21-as-fred-i-want-to-play-with-a-beautiful-board)

# Done
- [x] Add curses
- [x] Print board
- [x] User interactions

# Result

example

# Would be good to have (but don't have time)

- [ ] Tests on curses
- [ ] Fallback if curses not supported by user's terminal
- [ ] Management of error when window resized smaller than board

### CODE REVIEW

File: src/tiles.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+cross_raw = '''\
```
Comment (zyhou) [inline src/tiles.py:1]: :+1:

File: src/cursesio.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+
+def print_player(stdscr, player, phase=1):
+    stdscr.addstr(f'\n\nPlayer {get_player(player)} ({get_symbol(player)})')
+    text = 'Press space to select a '
+    text += 'cube to move' if phase == 1 else 'destination for the cube'
+    stdscr.addstr(f'\n{text}')
+
+
+def get_attr(pos, movables, selected, picked):
```
Comment (Kmaschta) [inline src/cursesio.py]: This function name isn't very descriptive.
What is the attribute does is return?

File: src/play.py [hunk 3]
```diff
@@ -1,39 +1,44 @@
...
 import game.moves as Moves
-import playerio as Io
+import cursesio as Io
+from curses import wrapper
+import curses
 
 
-def launch_game():
+def init_curses():
+    # curses.nonl()
```
Comment (Kmaschta) [inline src/play.py]: Commented out code

File: src/play.py [hunk 4]
```diff
@@ -1,39 +1,44 @@
 import game.board as Board
 import game.success as Success
 import game.moves as Moves
-import playerio as Io
+import cursesio as Io
```
Comment (Kmaschta) [inline src/play.py]: Not sure, but `io` is a standard module.
May I suggest `ui`

File: src/cursesio.py (hunk 2 above)
Comment (HALLERPierre) [reply src/cursesio.py]: Renamed get_style

**Comment by zyhou** [comment]
Update gif and readme gif

**Review by zyhou — APPROVED** [review]

**Pull request merged**